python/pyspark/broadcast.py

```diff
@@ -175,27 +203,27 @@ def destroy(self, blocking=False):
         self._jbroadcast.destroy(blocking)
         os.unlink(self._path)
 
-    def __reduce__(self):
+    def __reduce__(self) -> Tuple[Callable[[int], T], Tuple[int]]:
```

Comment:
This signature seems to be wrong. Double checking the flow:

-  `_from_id` returns `_broadcastRegistry[bid]`
- `_broadcastRegistry` is  `_broadcastRegistry: Dict[int, "Broadcast[Any]"]`
-  So `_from_id` is either `Callable[[int],  Broadcast[T]]`, or if it doesn't type check, `Callable[[int], Broadcast[Any]]`